tests/test_api.py

```diff
@@ -16,8 +16,8 @@ def client():
 
 
 def test_direct_download_url_small_file(client):
-    small_s3_file = '76%2F2%2Ffiles%2Fderivative%2FScaffold%2FmouseColon_metadata.json'
-    r = client.get(f"/s3-resource/{small_s3_file}")
+    small_s3_file = '76/files/derivative/mouseColon_metadata.json'
```

Comment:
Do these paths not require the version in the path anymore?

Reply:
No, version has been removed for a long time. All paths from the portal have been updated a while back.

Reply:
correct, in order to get the older versions you must first call the rehydration service